docs/doc_tutorial/05_helloworld.rst

```diff
@@ -83,8 +83,7 @@ your own python module structure fairly quickly.
         def show(self):
             popup.Alert(self._message)
 
-4. Run the ``flare/bin/gen-files-json.py`` utility script in your module directory. This will generate a ``files.json``
-file, that will look like this:
+4. Create a ``files.json`` file in your module directory and add the following content:
```

Comment:
Indeed, this is okay. People shall use the script from `flare.py` in future, which must generally be reworked in this tutorial. `flare.py` is currently rewritten and generates `files.json` automatically on build, see #46.